### PR TITLE
[cloud-provider-aws] fix terraform scheme

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>

## Description

Add `user_data_replace_on_change` to lifecycle ignore_changes list.

## Why do we need it, and what problem does it solve?

After #1656 terraform scheme was changed.
```
Terraform will perform the following actions:
│ │ 
│ │   # module.master-node.aws_instance.master will be updated in-place
│ │   ~ resource "aws_instance" "master" {
...
...
...
│ │         tenancy                              = "default"
│ │         user_data                            = "de654ea41c34ef1c62f4e"
│ │       + user_data_replace_on_change          = false
│ │         vpc_security_group_ids               = [
│ │             "sg-0123123123123",
│ │             "sg-0444444444444",
│ │             "sg-5555555555555",
│ │         ]
...
...
...
```
We added parameter to scheme to avoid manual converge of clusters.
## Changelog entries

```changes
section: cloud-provider-aws
type: fix
summary: fix terraform scheme
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/aws'...

**Comment by RomanenkoDenys** [comment]
Wrong way, PR closed

File: candi/cloud-providers/aws/terraform-modules/master-node/main.tf [hunk 1]
```diff
@@ -83,7 +83,10 @@ resource "aws_instance" "master" {
 
   lifecycle {
     ignore_changes = [
+      # user_data changes on aws side
```
Comment (konstantin-axenov) [inline candi/cloud-providers/aws/terraform-modules/master-node/main.tf:86]: Can you provide more information?

**Review by konstantin-axenov — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.32

**Pull request merged**